eunkyung/controllers/postController.js

```diff
@@ -0,0 +1,39 @@
+const postService = require('../services/postService')
+
+
+const posting = async (req, res) => {
```

Comment:
함수명 수정하면 좋을 것 같습니다.
`게시글을 생성한다`
```suggestion
const createPost = async (req, res) => {
```

Reply:
In [eunkyung/services/postService.js](https://github.com/wecode-bootcamp-korea/42-backend-westagram-1/pull/11#discussion_r1091549108):

> @@ -0,0 +1,36 @@

+const postDao = require('../models/postDao')

+

postService 부분도 postController 부분 참고하셔서 함수명 수정하면 좋을 것 같습니다!